src/components/tl/sharp/tl_sharp.h

```diff
@@ -53,6 +53,7 @@ typedef struct ucc_tl_sharp_context_config {
     int                      context_per_team;
     int                      enable_lazy_group_alloc;
     int                      team_max_ppn;
+    int                      multi_channel;
```

Comment:
nit: I would personally call that variable `enable_multi_channel` or `use_multi_channel`

Reply:
I agree, enable_multi_channel is more intuitive

Reply:
fixed